spec/facter/resolvers/uname_spec.rb

```diff
@@ -27,6 +27,10 @@
     Facter::Resolvers::Uname.invalidate_cache
   end
 
+  after do
+    Facter::Resolvers::Uname.invalidate_cache
```

Comment:
suggest moving this into the the existing `after` block to fix the rubocop issue